### DISCUSSION TITLE
OKS_SIGMA

### DISCUSSION THREAD

**Discussion by ItayCoifman** [ultralytics/ultralytics#3307] [Ideas]
`OKS_SIGMA` is used to calculate the loss for pose model, its a hard coded variable found on `ultralytics.yolo.utils.metrics`.
Basically this hard coded variable is the weights of the keypoints for the pose model. 
This variable is a crucial hyperparameter for loss calculation in pose models, I think it will be great if this parameter would be accessible from conif.yaml
Thanks :)

**Comment by glenn-jocher** [comment]
@ItayCoifman hello! Thank you for your valuable suggestion regarding the `OKS_SIGMA` parameter for pose model loss calculation. Making hyperparameters like this configurable through the `config.yaml` file is indeed a good idea to provide more flexibility and control to the users. We appreciate your input and will consider this for future updates. Keep an eye on the repository for any changes, and feel free to contribute or provide more feedback. Your engagement helps make our tools better for everyone! 🚀🤝
